pkg/nfd-master/updater-pool.go

```diff
@@ -85,16 +87,48 @@ func (u *nodeUpdaterPool) runNodeUpdater(queue workqueue.RateLimitingInterface)
 	u.wg.Done()
 }
 
-func (u *nodeUpdaterPool) start(parallelism int) {
+func (u *updaterPool) processNodeFeatureGroupUpdateRequest(ngfQueue workqueue.RateLimitingInterface) bool {
```

Comment:
Rebase on the latest master and replicate that we now have in `processNodeUpdateRequest()`. On top of my head at least:

- `cli` (kubernetes client) is an arg to this method
- check existence of NFG and pass cli and that object as args to `u.nfdMaster.nfdAPIUpdateNodeFeatureGroup`
- on errors retry indefinitely (unless notfound)

Reply:
done